src/molecules/Modal/modal.module.scss

```diff
@@ -0,0 +1,72 @@
+@import 'assets/stylesheets/global';
+
+$modal-standard-max-width:                  100%;
+$modal-wide-max-width:                      100%;
+$modal-image-max-width:                     rem-calc(300px);
+$modal-border-color:                        color('neutral-5');
+$modal-header-background-color:             color('neutral-6');
+$modal-header-color:                        color('neutral-2');
+$modal-body-background-color:               color('neutral-8');
+$modal-body-color:                          color('neutral-2');
+$modal-body-subtitle-color:                 color('neutral-3');
+$modal-tab-border-color:                    color('neutral-5');
+$modal-tab-background-color:                color('neutral-7');
+$modal-tab-active-background-color:         color('neutral-8');
+$modal-tab-active-color:                    color('neutral-1');
+$robust-card-mobile-top-panel:              color('neutral-7');
+$robust-card-mobile-top-panel-border-color: color('neutral-5');
+$robust-card-mobile-top-panel-name-color:   color('neutral-3');
+
+.modal {
+  margin: ru(4.25) auto 0;
+
+  &.simple {
+    max-width: rem-calc(728);
+  }
+}
+
+.dialog {
+  position: relative;
+  z-index: 9999;
+  max-width: 100%;
+  margin: auto;
+  border-radius: ru(.25);
+  border: 1px solid color('neutral-5');
+  font-family: 'adelle-sans', Helvetica, Arial, sans-serif;
```

Comment:
@jmcolella You shouldn't be setting this directly. Please use the text mixin to maintain our standards / branding.